### DISCUSSION TITLE
How to disable migration log in test

### DISCUSSION THREAD

**Discussion by kevindavee** [mikro-orm/mikro-orm#4067] [Q&A]
Hi, I'm using MikroORM and NestJS. I run the migration on `onModuleInit` of `app.module.ts`

```
async onModuleInit(): Promise<void> {
    await this.orm.getMigrator().up();
  }
```

When running e2e test, I got flooded with `console.log` statement of Migrations X is executed. It spams the console and it's hard to see the test result. I can't find how to disable the console.log

Anyone can help?

**Comment by kevindavee** [comment]
@B4nan can you help with this?

**Reply by alexandregrm** [reply]
It's a workaround but you can override the logger with empty one : 

```
return MikroORM.init({
  debug: true,
  logger: () => {},
});
```

[Related documentation](https://mikro-orm.io/docs/next/logging#using-a-custom-logger)
